monkey/infection_monkey/system_info/windows_info_collector.py

```diff
@@ -46,16 +47,33 @@ def get_info(self):
         return self.info
 
     def get_installed_packages(self):
-        LOG.info('getting installed packages')
-        self.info["installed_packages"] = os.popen("dism /online /get-packages").read()
-        self.info["installed_features"] = os.popen("dism /online /get-features").read()
+        LOG.info('Getting installed packages')
+        system32_dir = os.environ["windir"] + "\\system32"
+        go_to_system32_dir_cmd = f"cd {system32_dir}"  # dism only works in %windir%\system32 (with admin privileges)
```

Comment:
Says who? Docs: `DISM comes with Windows 10, in the c:\windows\system32 folder, but you can run DISM in the command prompt from any location.` I tested myself and was able to run from my user dir. What am I missing?

Reply:
For some reason, my test machine didn't have C:\Windows\system32 to the PATH variable by default, so it wouldn't recognise the command. I felt it would be safe to change the directory first but on second thought, this is unexpected behaviour, so I'll change it back.

Reply:
Makes sense. All windows have a system32 folder but it seems possible not to have it in the path. I don't mind either way, just wanted to understand.